bentoml/deployment/sagemaker/__init__.py

```diff
@@ -259,6 +258,57 @@ def init_sagemaker_project(sagemaker_project_dir, bento_path):
     return sagemaker_project_dir
 
 
+def _create_sagemaker_model(
+    sagemaker_client, bento_name, bento_version, ecr_image_path, api_name
+):
+    execution_role_arn = get_arn_role_from_current_aws_user()
+    model_name = create_sagemaker_model_name(bento_name, bento_version)
+
+    sagemaker_model_info = {
+        "ModelName": model_name,
+        "PrimaryContainer": {
+            "ContainerHostname": model_name,
+            "Image": ecr_image_path,
+            "Environment": {
+                "API_NAME": api_name,
+                "BENTO_SERVER_TIMEOUT": config().get('apiserver', 'default_timeout'),
+                "BENTO_SERVER_WORKERS": config().get(
+                    'apiserver', 'default_gunicorn_workers_count'
+                ),
+            },
+        },
+        "ExecutionRoleArn": execution_role_arn,
+    }
+
+    logger.debug("Creating sagemaker model %s", model_name)
+    create_model_response = sagemaker_client.create_model(**sagemaker_model_info)
+    logger.debug("AWS create model response: %s", create_model_response)
+    return model_name
+
+
+def _create_sagemaker_endpoint_config(
+    sagemaker_client, model_name, bento_name, bento_version, sagemaker_config
+):
+    production_variants = [
+        {
+            "VariantName": generate_aws_compatible_string(bento_name),
+            "ModelName": model_name,
+            "InitialInstanceCount": sagemaker_config.instance_count,
+            "InstanceType": sagemaker_config.instance_type,
+        }
+    ]
+    endpoint_config_name = create_sagemaker_endpoint_config_name(
```

Comment:
can we rename those two functions to verb `get` and mark as private in the module? for example:
```
def _get_sagemaker_model_name(bento_name, bento_version):
   ...

def _get_sagemaker_endpoint_config_name(bento_name, bento_version)
   ...
```
or 
```
def _sagemaker_model_name(bento_name, bento_version):
   ...

def _sagemaker_endpoint_config_name(bento_name, bento_version)
   ...
```